_docs/_latest/concepts/best-practices.md

```diff
@@ -113,4 +272,126 @@ whitelist IP addresses.
 authorized networks.**
 
 By using SSL connections, you prevent third parties from seeing the data that's transferred
-between client and server over insecure networks.
\ No newline at end of file
+between client and server over insecure networks.
+
+## Event Logging and Resource Monitoring
+
+**Stackdriver Account per Environment**
+
+Best practice is to have a Stackdriver account per environment (DEV, TEST, &
+PROD). Projects can be added to these Stackdriver accounts as a Monitored
+project. It is recommended to initialize the Stackdriver account in a separate
+project - to allow for independent management of the IAM policies.
+
+**Logging Retention**
```

Comment:
I don't think we can check for this in Forseti meaningfully? Or do you have a suggestion? I think logs do have 'common' names, but can we even determine the logging bucket AND determine if there's a secondary bucket (with e.g. coldline settings) that has some but not all..... 

I'd like to implement in Forseti but it doesn't sound easy..